cfgov/ask_cfpb/models/pages.py

```diff
@@ -186,14 +186,11 @@ def get_context(self, request, *args, **kwargs):
 
 
 class SecondaryNavigationJSMixin:
-    """A page mixin that adds navigation JS for English pages."""
+    """A page mixin that adds navigation JS."""
 
     @property
     def page_js(self):
-        js = super().page_js
-        if self.language == "en":
```

Comment:
This looks very intentional to have only on english pages. Any idea why it was originally written like this?

Reply:
This was added in #2909. At that time, the portal search pages had different English and Spanish templates (see screenshots on the PR, or code [here](https://github.com/cfpb/consumerfinance.gov/pull/2909/files#diff-91718ae187b630e80b3fc7f1e6cb188b2b13ce069a6815be41fdad200e04050dR76)). Only the English pages had the secondary nav sidebar, so only they needed the JS. When the Spanish pages were migrated to the same template this code must have been overlooked.